numpy/matlib.py

```diff
@@ -210,7 +213,7 @@ def eye(n,M=None, k=0, dtype=float):
             [ 0.,  0.,  0.]])
 
     """
-    return asmatrix(np.eye(n, M, k, dtype))
+    return asmatrix(np.eye(n, M=M, k=k, dtype=dtype, order=order))
```

Comment:
Switching from positional to keyword is a bit of editorializing by me and I'm happy to revert if desired.

Reply:
Keywords are slower, which is why you'll often see them omitted.

Reply:
On the other hand, the more things to deter users from `matlib`, the better!